HACKING.md

```diff
@@ -17,6 +17,26 @@ On Debian/Ubuntu:
 
     $ sudo apt install nodejs
 
+
+Creating VM images locally (not necessary for running tests) needs the
+following:
+    
+    $ sudo apt-get install virtinst
+
+For running integration tests, the following dependencies are required:
+
+    $ sudo apt-get install curl expect xz-utils rpm libvirt-daemon-system libvirt-daemon-driver-storage-core \
+        libvirt-daemon-driver-qemu libvirt-clients python3-libvirt
+        python3-pyyaml python3-flake8 \chromium-browser
+
+Updating the `node_modules` (in case you need to modify `package.json`)
+requires npm to be installed.
+
+    $ sudo apt-get install npm
```

Comment:
npm is a dependent package of nodejs which is already mentioned as dependency on the top of the document  https://github.com/cockpit-project/cockpit/blame/main/HACKING.md#L18

Reply:
This is also not true any more. `tools/node-modules install` runs in a podman container, not with local `npm`.

Reply:
I see, I'll have it removed from the document